utils/use_require_helpers.js

```diff
@@ -21,17 +32,29 @@ module.exports = {
         appWriter: (base_out_path, out_path) => {
             var browserify = require('browserify');
             var b = browserify(path.join(base_out_path, 'app/ui.js'), {});
-            b.bundle().pipe(fs.createWriteStream(out_path));
-            return `<script src="${path.relative(base_out_path, out_path)}"></script>`;
+            return new Promise((resolve, reject) => {
+                b.bundle((err, buf) => {
+                    if (err) return reject(err);
+                    resolve(buf);
+                });
+            })
+            .then((buf) => {
```

Comment:
`.then((buf) => writeFile(out_path, buff))`

No need to use blockform for single-expression returns.  It's more readable without the block

Reply:
Will fix.